src/standalone/Snow/Snow.jl

```diff
@@ -24,16 +54,249 @@ Base.@kwdef struct SnowParameters{FT <: AbstractFloat, PSE}
+    ρcD_g::FT
     "Clima-wide parameters"
     earth_param_set::PSE
 end
 
-include("./snow_parameterizations.jl")
+"""
+   SnowParameters{FT}(Δt;
+                      ρ_snow = FT(200),
+                      z_0m = FT(0.0024),
+                      z_0b = FT(0.00024),
+                      α_snow = FT(0.8),
+                      ϵ_snow = FT(0.99),
+                      θ_r = FT(0.08),
+                      Ksat = FT(1e-3),
+                      κ_ice = FT(2.21),
+                      ρcD_g = FT(3.553e5),
+                      earth_param_set::PSE) where {FT, PSE}
+
+An outer constructor for `SnowParameters` which supplies defaults for
+all arguments but `earth_param_set`.
+"""
+function SnowParameters{FT}(
+    Δt;
+    ρ_snow = FT(200),
+    z_0m = FT(0.0024),
+    z_0b = FT(0.00024),
+    α_snow = FT(0.8),
+    ϵ_snow = FT(0.99),
+    θ_r = FT(0.08),
+    Ksat = FT(1e-3),
+    κ_ice = FT(2.21),
+    ρcD_g = FT(3.553e5),
+    earth_param_set::PSE,
+) where {FT <: AbstractFloat, PSE}
+    return SnowParameters{FT, PSE}(
+        ρ_snow,
+        z_0m,
+        z_0b,
+        α_snow,
+        ϵ_snow,
+        θ_r,
+        Ksat,
+        κ_ice,
+        Δt,
+        ρcD_g,
+        earth_param_set,
+    )
+end
+
+Base.broadcastable(ps::SnowParameters) = tuple(ps)
+
+"""
+    struct SnowModel{
+        FT,
+        PS <: SnowParameters{FT},
+        ATM <: AbstractAtmosphericDrivers{FT},
+        RAD <: AbstractRadiativeDrivers{FT},
+        D,
+    } <: AbstractSnowModel{FT}
+
+A container/type for the bulk snow model, based on the UEB snow model
+of Tarboton et al. (1995) and Tarboton and Luce (1996).
+"""
+struct SnowModel{
+    FT,
+    PS <: SnowParameters{FT},
+    ATM <: AbstractAtmosphericDrivers{FT},
+    RAD <: AbstractRadiativeDrivers{FT},
+    D,
+} <: AbstractModel{FT}
+    "Parameters required by the snow model"
+    parameters::PS
+    "The atmospheric drivers: Prescribed or Coupled"
+    atmos::ATM
+    "The radiation drivers: Prescribed or Coupled"
+    radiation::RAD
+    "The domain of the model"
+    domain::D
+end
+
+function SnowModel(;
+    parameters::SnowParameters{FT, PSE},
+    domain::ClimaLand.Domains.AbstractDomain,
+    atmosphere::ATM,
+    radiation::RAD,
+) where {FT, PSE, ATM, RAD}
+    args = (parameters, atmosphere, radiation, domain)
+    SnowModel{FT, typeof.(args)...}(args...)
+end
+
+
+prognostic_types(::SnowModel{FT}) where {FT} = (FT, FT)
+prognostic_vars(::SnowModel) = (:S, :U)
+
+prognostic_domain_names(::SnowModel) = (:surface, :surface)
+
+auxiliary_types(::SnowModel{FT}) where {FT} = (
+    FT,
+    FT,
+    FT,
+    NamedTuple{(:lhf, :shf, :vapor_flux, :r_ae), Tuple{FT, FT, FT, FT}},
+    FT,
+    FT,
+    FT,
+    FT,
+    FT,
+)
+
+auxiliary_domain_names(::SnowModel) = (
+    :surface,
+    :surface,
+    :surface,
+    :surface,
+    :surface,
+    :surface,
+    :surface,
+    :surface,
+    :surface,
+)
+
+auxiliary_vars(::SnowModel) = (
```

Comment:
same here, comment with full names? 
(note that these will be given full names in ClimaDiagnostics outputs - but it can be useful to read the code)

Reply:
I add this is in too - good call